pkg/backporter/backporter.go

```diff
@@ -438,16 +443,16 @@ func getClonesTemplateData(bugID int, client bugzilla.Client, allTargetVersions
 	// find the major release of the clone targeting the first release
 	for _, clone := range clones {
 		var err error
-		if len(clone.TargetRelease) > 0 {
-			firstClone, err = getMajorRelease(clone.TargetRelease[0])
+		if len(clone.TargetRelease) > 0 && clone.TargetRelease[0] != "---" {
```

Comment:
The fact that we're checking that the release is not empty or set to `"---"` over N different places is an opportunity to DRY.

Reply:
I am guessing you mean to move `len(clone.TargetRelease) > 0 && clone.TargetRelease[0] != "---"` this into a isTargetReleaseValid function. lmk if that's not what you meant

Reply:
yeah and ideally return an error if that happens rather than just ignoring the bug

Reply:
Not only to not ignore it, but also in your code maybe

```go
func releaseUnset(bug bugzilla.Bug) bool {
    return bug.TargetRelease == "" || bug.TargetRelease == "---"
}

// ...
if releaseUnset(clone) {
    return fmt.Errorf("could not ... for bug %d, target release field was unset", clone.Id)
}
```